<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Thermal_Hydraulics</class>
 <widget class="QMainWindow" name="Thermal_Hydraulics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>665</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thermal-Hydraulics</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" alignment="Qt::AlignHCenter">
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">selection-background-color: qconicalgradient(cx:0.5, cy:0.5, angle:0, stop:0 rgba(0, 241, 254, 255), stop:1 rgba(255, 255, 255, 255));</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="maximumSize">
          <size>
           <width>770</width>
           <height>450</height>
          </size>
         </property>
         <property name="title">
          <string>Radial Temperatre Distribution</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="2" column="0">
           <widget class="QGroupBox" name="groupBox_10">
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_12">
             <item row="3" column="2">
              <widget class="QLineEdit" name="lineEdit_26"/>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_27">
               <property name="text">
                <string>Conductivity of the cladding</string>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <widget class="QLineEdit" name="lineEdit_28"/>
             </item>
             <item row="6" column="2">
              <widget class="QLineEdit" name="lineEdit_29"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_25">
               <property name="text">
                <string>Width of the cladding</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_26">
               <property name="text">
                <string>Conductivity of the fuel</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="lineEdit_24"/>
             </item>
             <item row="7" column="2">
              <widget class="QLineEdit" name="lineEdit_31"/>
             </item>
             <item row="8" column="2">
              <widget class="QLineEdit" name="lineEdit_32"/>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="label_30">
               <property name="text">
                <string>Temperature at inner surface of cladding</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_28">
               <property name="text">
                <string>Heat flux</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLineEdit" name="lineEdit_25"/>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_29">
               <property name="text">
                <string>Power density</string>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QLineEdit" name="lineEdit_27"/>
             </item>
             <item row="9" column="0">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="label_31">
               <property name="text">
                <string>Temperature at the outer surface of cladding</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Heat transfer coefficient</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLineEdit" name="lineEdit_3"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>Radius of the fuel</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QGroupBox" name="groupBox_6">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>160</height>
             </size>
            </property>
            <property name="title">
             <string>Radial Discritisation</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_8">
             <item row="0" column="0">
              <widget class="QGroupBox" name="groupBox_7">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>100</height>
                </size>
               </property>
               <property name="title">
                <string>Fuel</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_9">
                <item row="1" column="1">
                 <widget class="QLineEdit" name="lineEdit_16"/>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="label_17">
                  <property name="text">
                   <string>Rf</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_15">
                  <property name="toolTip">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                  <property name="text">
                   <string>Rc</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLineEdit" name="lineEdit_17"/>
                </item>
                <item row="0" column="2">
                 <widget class="QLabel" name="label_16">
                  <property name="text">
                   <string>Step</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QLineEdit" name="lineEdit_15"/>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QGroupBox" name="groupBox_8">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>110</height>
                </size>
               </property>
               <property name="title">
                <string>Clad</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_10">
                <item row="1" column="0">
                 <widget class="QLineEdit" name="lineEdit_18"/>
                </item>
                <item row="0" column="2">
                 <widget class="QLabel" name="label_20">
                  <property name="text">
                   <string>Step</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QLineEdit" name="lineEdit_19"/>
                </item>
                <item row="1" column="2">
                 <widget class="QLineEdit" name="lineEdit_20"/>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_18">
                  <property name="text">
                   <string>R(inner)</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="label_19">
                  <property name="text">
                   <string>R(outer)</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QGroupBox" name="groupBox_9">
               <property name="title">
                <string>Coolant</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_11">
                <item row="1" column="1">
                 <widget class="QLineEdit" name="lineEdit_22"/>
                </item>
                <item row="1" column="0">
                 <widget class="QLineEdit" name="lineEdit_21"/>
                </item>
                <item row="0" column="2">
                 <widget class="QLabel" name="label_21">
                  <property name="text">
                   <string>Step</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLabel" name="label_23">
                  <property name="text">
                   <string>R(outer)</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QLineEdit" name="lineEdit_23"/>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_22">
                  <property name="text">
                   <string>R(inner)</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>339</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>Axial Temperatre Distribution</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="0" colspan="3">
           <widget class="QGroupBox" name="groupBox_5">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>300</height>
             </size>
            </property>
            <property name="title">
             <string/>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="3" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Mass flow rate</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Exit temperature</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="lineEdit_12"/>
             </item>
             <item row="7" column="1">
              <widget class="QLineEdit" name="lineEdit_14"/>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Heat transfer coefficient</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Specific heat at constant pressure</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLineEdit" name="lineEdit_13"/>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Heated perimeter</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Core height/Pin cell</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QLineEdit" name="lineEdit_10"/>
             </item>
             <item row="3" column="1">
              <widget class="QLineEdit" name="lineEdit_9"/>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEdit_2"/>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_7"/>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Height including the reflector</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Inlet temperature</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLineEdit" name="lineEdit_8"/>
             </item>
            </layout>
           </widget>
          </item>
          <item row="0" column="0" colspan="3">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>110</height>
             </size>
            </property>
            <property name="title">
             <string>Axial Discritisation</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Minus Z</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Step</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLineEdit" name="lineEdit_4"/>
             </item>
             <item row="1" column="2">
              <widget class="QLineEdit" name="lineEdit_6"/>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEdit_5"/>
             </item>
             <item row="0" column="1">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Plus Z</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" rowspan="2">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="minimumSize">
          <size>
           <width>339</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>110</height>
          </size>
         </property>
         <property name="title">
          <string>Thermal-Hydraulic</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_13">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Total channel power</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_12">
         <property name="title">
          <string>Control</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_14">
          <item row="3" column="1">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Plot</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Run</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>Case study of radial</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Case study of axial</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1082</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Exit</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit_2"/>
   </widget>
   <widget class="QMenu" name="menuClear">
    <property name="title">
     <string>Clear</string>
    </property>
    <addaction name="actionClear_2"/>
   </widget>
   <addaction name="menuExit"/>
   <addaction name="menuClear"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionClear_2">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
